sql/moz-fx-data-shared-prod/telemetry_derived/firefox_android_clients_v1/query.sql

```diff
@@ -0,0 +1,482 @@
+-- Query first observations for Firefox Android Clients.
+-- Proposal:
+-- https://docs.google.com/document/d/12bj4DhCybelqHVgOVq8KJlzgtbbUw3f68palNrv-gaM/edit#
+CREATE TEMP FUNCTION get_first_not_null_value(
```

Comment:
If we move this to mozfun, it would be easy to write some tests against it (as compared to writing tests for the entire sql query which is much more tedious)

Reply:
Sound like a good idea, then we wouldn't need the table schema. Let me have a look,

Reply:
Moved to `mozfun.norm.get_earliest_value`.